### ISSUE TITLE
Results not collected in NVDA Chrome automated testing

### ISSUE THREAD

**Issue by ccanash** [w3c/aria-at-app#1060]
Using automated testing for radio button, the folllowing issues were found: 

1.  For tests 1 through 4 for tab and shift+tab with focus mode the bot recorded no output
2. For tests 5 through 6 bot recorded no output for up arrow with browse mode and shift tab with focus mode
3. For test 11 bot reported no output for space with focus mode
4. For tests 12 and 13 bot reported no output for both cases

**Comment by gnarf** [comment]
Recorded a run of the suite on my local VM https://www.youtube.com/watch?v=SO08WD36joU - haven't gone through and analyzed  the specific test failures, but this video might help whomever gets assigned to it.

**Comment by gnarf** [comment]
The main problem is failing press key events on NVDA driver.

See https://github.com/Prime-Access-Consulting/nvda-at-automation/issues/49 for general issue description. Included links to recording of testing VM running tests and the some additional logging from nvda.log 

## Test 1 - After 'Tab' (focus mode on)
https://youtu.be/SO08WD36joU?si=Knzq_1FPFMZ6Sgs4&t=71
```
executing gesture "tab"
error executing gesture tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 2 - After 'Shift+Tab' (focus mode on)
https://youtu.be/SO08WD36joU?si=UATg9e8k0Exis5JA&t=133
```
executing gesture "shift+tab"
error executing gesture shift+tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 3 - After 'Tab' (focus mode on) 
https://youtu.be/SO08WD36joU?si=Y6yVZ7zHzxrtRn8k&t=200
```
executing gesture "tab"
error executing gesture tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 4 - After 'Shift+Tab' (focus mode on)
https://youtu.be/SO08WD36joU?si=_rQ7_o9DATs-kIzX&t=270
```
executing gesture "shift+tab"
error executing gesture shift+tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 5 - After 'Up Arrow' (browse mode on)
https://youtu.be/SO08WD36joU?si=VMwd80JhLey8pn21&t=299
```
executing gesture "up"
invalid gesture "up
```

## Test 5 - After 'Shift+Tab' (focus mode on)
https://youtu.be/SO08WD36joU?si=NtnORcmNWKEtCDxD&t=331
```
executing gesture "shift+tab"
error executing gesture shift+tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 6 - After 'Down Arrow' (browse mode on)
https://youtu.be/SO08WD36joU?si=RkTMJuOdykCxMYyy&t=361
```
executing gesture "down"
invalid gesture "down
```

## Test 6 - After 'Tab' (focus mode on)
https://youtu.be/SO08WD36joU?si=-CC058_AyWDIJP0L&t=392
```
executing gesture "tab"
error executing gesture tab:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 11 - After 'Space' (focus mode on)
https://youtu.be/SO08WD36joU?si=Mk0iYyUs-Yzb0f-j&t=581
```
executing gesture "space"
error executing gesture space:
Traceback (most recent call last):
  File "C:\Users\User\Desktop\2023.3.3\userConfig\addons\CommandSocket\globalPlugins\CommandSocket\handler.py", line 91, in _handle_press_keys_command
    inputCore.manager.executeGesture(gesture)
  File "inputCore.pyc", line 574, in executeGesture
inputCore.NoInputGestureAction
```

## Test 12 / 13 - After '* Arrow' (focus mode on)

There are 4 tests in a row here, all 4 have the same general error (arrow keys missing gestures)

https://youtu.be/SO08WD36joU?si=ihRGF9BH0Ox5nkQg&t=590

```
executing gesture "down"
invalid gesture "down
```

```
executing gesture "right"
invalid gesture "right
```

```
executing gesture "up"
invalid gesture "up
```

```
executing gesture "left"
invalid gesture "left
```

**Comment by ccanash** [comment]
Closing as addressed by [PR 4](https://github.com/Prime-Access-Consulting/nvda-at-automation/issues/49)
